src/config/index.js

```diff
@@ -91,6 +91,7 @@ const defaultConfig = {
   updateChannel: 'stable',
   traefikImage: 'traefik:latest',
   traefikName: 'exoframe-traefik',
+  traefikPath: path.join(os.homedir(), '.exoframe', 'traefik'),
```

Comment:
This won't really work though, right? If you are already running in container - this will point to the folder inside of that container.

Reply:
Yeah, you are right. Didn't think about that. Glad you mention it.

We would need to mount the volume to both exoframe-server and exoframe-traefik.
While ensuring both "outer" binds match.

What about ``dockerode.listVolumes()``, pick the ``/path/to/exoframe-folder:/root/.exoframe`` volume and filter out the inner bind ``:/root/.exoframe``

Then we are left with ``/path/to/exoframe-folder`` which will be appended ``/traefik`` folder.

Do you think that would work?

Seems to me as plausible solution ensuring manipulation in both exoframe-server and exoframe-traefik

Reply:
Yeah, that was my initial thought. This is also likely the easiest way to implement it 🤔 
We also need to keep in mind that the server might be executed without container.